Makefile

```diff
@@ -26,8 +26,14 @@ docker-importer:
 	docker-compose run --rm importer
 	chmod a+rw docker/trex_cache
 
-test:
-	./scripts/test_on_monaco.sh
+docker/imposm/import/monaco.osm.pbf:
+	wget http://download.geofabrik.de/europe/monaco-latest.osm.pbf --no-verbose -O $@
+
+#to get the bbox :  osmconvert --out-statistics docker/imposm/import/monaco.osm.pbf | egrep 'lon |lat ' | cut -d ' ' -f 3 | tr '\n' ' ' | sed -E "s/([^ ]+) ([^ ]+) ([^ ]+) ([^ ]+)/\1,\3,\2,\4/"
+test-monaco: docker/imposm/import/monaco.osm.pbf docker-importer
```

Comment:
I find this is clearer to use a separate script for the tests over a makefile (but this is fine for me for now, we may change back if we add more complexity to the tests)